eth/scripts/1_l2_setup_bridge_contracts.js

```diff
@@ -99,18 +101,31 @@ async function initializePool(config, pool, lpAddress, bazero, owner) {
   });
 }
 
+function getWazeroAddress() {
+  const { alephTokens } = JSON.parse(
+    fs.readFileSync("../azero/l2_addresses.json", {
+      encoding: "utf8",
+      flag: "r",
+    }),
+  );
+
+  return alephTokens[0].address;
+}
+
 async function most_add_bazero_pair(most, bazeroAddress) {
   const bazeroAddressBytes = ethers.zeroPadValue(
     ethers.getBytes(bazeroAddress),
     32,
   );
-  /// TODO: switch this for actual address of wrapped azero on L1
-  const destBazeroAddressBytes = bazeroAddressBytes;
+  const wazero = getWazeroAddress();
+  const azeroTokenAddressBytes = u8aToHex(
+    new Keyring({ type: "sr25519" }).decodeAddress(wazero),
+  );
 
   console.log(
-    `Adding \`Bazero\` pair to the most ${bazeroAddressBytes} <-> ${destBazeroAddressBytes}`,
+    `Adding \`Bazero\` pair to the most ${bazeroAddressBytes} <-> ${azeroTokenAddressBytes}`,
   );
-  await most.addPair(bazeroAddressBytes, destBazeroAddressBytes, false);
+  await most.addPair(bazeroAddressBytes, azeroTokenAddressBytes, false);
```

Comment:
Order of arguments  does not matter right for MOST logic right?

Reply:
it matters. Take a look at the contracts addPair implementation, you will see it not a 'two way' but one way